src/Slices/ResourceDiscovery/Core/Query.ts

```diff
@@ -31,6 +31,7 @@ export interface Manifest {
 
 export interface DiscoveredResource {
   discovered_resource_id: string;
+  managed_resource_uri: string | null;
```

Comment:
just out of curiosity, in case of no uri won't that value be undefined? I'm not sure, but often that's the case with the responses from the backend

Reply:
No, it is null :D